Gateway/Request/CaptureRequest.php

```diff
@@ -46,12 +46,13 @@ public function build(array $buildSubject)
         $payment = $paymentDataObject->getPayment();
         $transactionId = $payment->getAdditionalInformation(self::TRANSACTION_ID) ?:
             $payment->getAdditionalInformation(self::CHARGE_ID);
+        $countryCode = $payment->getAdditionalInformation(self::COUNTRY_CODE)?: 'USA';
```

Comment:
can we use default_country_code enum instead of USA

Reply:
added an enum and changed to: `$countryCode = $payment->getAdditionalInformation(self::COUNTRY_CODE) ?: self::DEFAULT_COUNTRY_CODE;`